src/lightmap.cpp

```diff
@@ -46,6 +46,7 @@
 #include "vpart_range.h"
 #include "weather.h"
 #include "weather_type.h"
+#include <units_utility.h>
```

Comment:
Please use quotes (`#include "units_utility.h"`) or it doesn't compile, and insert it after `#include "units.h"`, so headers are listed in lexicographical order.